README.md

```diff
@@ -78,27 +176,29 @@ So here are three default placeholders for your file system:
 
 - `~` Your **home** directory
 - `.` The **current** directory
-- `..` The directory in which your current directory is contained—referred to as the "**parent**"
-directory.
+- `..` The directory in which your current directory is contained—referred to
+as the "**parent**" directory.
 
 You can supply any path to the `cd` command to nkellyegreenegate to that location.
 
 ## Demonstrate Manipulating Files in the Command Line
 
 ### Paths in Shell
 
-The path supplied to the `cd` command, for example `/Users/kellyegreene`, is known as an absolute
-path.
+The path supplied to the `cd` command, for example `/Users/kellyegreene`, is
+known as an absolute path.
 
 Systems can use either *absolute* or *relative* paths.
 
-An absolute path is a path that points to the same location on the file system regardless of the
-working directory. They start with `/` ("forward slash") because that is the root of your file system.
+An absolute path is a path that points to the same location on the file system
+regardless of the working directory. They start with `/` ("forward slash") because
+that is the root of your file system.
 
 This is an absolute path: `/Users/kellyegreene`.
 
-A relative path is a path **relative** to the working directory of the user or application, so the
-full absolute path will not have to be given. They start with the name of a directory or a file.
+A relative path is a path **relative** to the working directory of the user or
```

Comment:
My edit immediately above creates symmetry to this paragraph which is very good.